### DISCUSSION TITLE
Animate, output only to a window

### DISCUSSION THREAD

**Discussion by SullyPanda76cl** [ImageMagick/ImageMagick#6182] [Help]
### ImageMagick version

7.1.1-4

### Operating system, version and so on

W11

### Description

Hi... in searching of an app that will show my loved ones photos on a fixed window, I reached ImageMackick, via this suggestion:
[SuperUser](https://superuser.com/questions/1650108/how-to-play-an-image-slideshow-in-a-windowed-mode-non-fullscreen-on-linux)

But on trying:
`animate -pause 2 *.jpg
`
It just gives me the info usage output...

Perhaps is it different in v7?
Thanks!

**Comment by snibgo** [comment]
`animate` runs on X-Windows, not Microsoft Windows. I have heard that X-Windows can be installed on Microsoft Windows, but I haven't done so.
